src/Http/Routing/src/ValidationEndpointFilterFactory.cs

```diff
@@ -53,14 +41,15 @@ public static EndpointFilterDelegate Create(EndpointFilterFactoryContext context
 
             if (options.TryGetValidatableParameterInfo(parameters[i], out var validatableParameter))
             {
+                validatableParameters ??= new();
                 validatableParameters.Add(new ValidatableParameterEntry(
```

Comment:
Can use more C# gimmick:
```suggestion
                (validatableParameters ?? []).Add(new ValidatableParameterEntry(
```

Reply:
@copilot Let's take advantage of collection expressions here.

```suggestion
                validatableParameters ??= [];
                validatableParameters.Add(new ValidatableParameterEntry(
```